pkg/controller/sparkapplication/submission.go

```diff
@@ -62,7 +62,9 @@ func runSparkSubmit(submission *submission) (bool, error) {
 
 	cmd := execCommand(command, submission.args...)
 	glog.V(2).Infof("spark-submit arguments: %v", cmd.Args)
-	if _, err := cmd.Output(); err != nil {
+	output, err := cmd.Output();
+	glog.V(3).Infof("spark-submit output: %s", string(output))
```

Comment:
This might be useful in client mode and when spark-submit prints useful output in cluster mode eg. when using kerberos.